pkg/conversion/error.go

```diff
@@ -54,6 +54,10 @@ type missingKindErr struct {
 	data string
 }
 
+func NewMissingKindErr(data string) error {
+	return &missingKindErr{data}
+}
+
 func (k *missingKindErr) Error() string {
```

Comment:
Yeah, we need to formalize conversion errors.  Was figuring post 1.0 we'd come back to the runtime/conversion Scheme discussion and formalize this (for smartobjects or whatever we want to call it).

----- Original Message -----
> > @@ -54,6 +54,10 @@ type missingKindErr struct {
> >  	data string
> >  }
> >  
> > +func NewMissingKindErr(data string) error {
> > +	return &missingKindErr{data}
> > +}
> > +
> >  func (k *missingKindErr) Error() string {
> 
> /grumpycat error types should be public for introspection. You don't need to
> fix in this PR though.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/7490/files#r29354210